src/clj/harja/palvelin/integraatiot/api/tyokoneenseuranta.clj

```diff
@@ -58,11 +58,30 @@
      :urakkaid         urakka-id
      :tehtavat         (arrayksi db (get-in havainto [:havainto :suoritettavatTehtavat]))}))
 
+(defn- filtterin-agressiivisuus
+  "Tavoitteena on vähentää reittipisteiden määrää progressiivisesti."
+  [koko]
+  (cond
+    (< koko 100) 1
+    (< koko 250) 2
+    (< koko 1000) 4
+    (< koko 2000) 8
+    :else 10))
+
 (defn- tallenna-seurantakirjaus-viivageometriana [_ data kayttaja db]
   (doseq [havainto (:havainnot data)]
-    (let [urakka-id (get-in havainto [:havainto :urakkaid])]
-      (when urakka-id (validointi/tarkista-jarjestelma-urakka-ja-kayttaja db urakka-id kayttaja))
-      (tallenna-tyokoneen-reitti db data havainto urakka-id)))
+    (let [;; Seurataan yksittäisen havainnon käsittelyn aikaa
+          alkuaikams (clj-time.coerce/to-long(clj-time.core/now))
```

Comment:
puuttuuko välilyönti to-long jälkeen? Usein on käytetty myös harja.pvm/nyt funktiota, sillä 100+ käyttöä

Reply:
Copypastevirhe. Eli puuttuuhan sieltä. Vaihdoin myös pvm/nyt funkkariin